config.def.h

```diff
@@ -67,6 +67,9 @@ static const int THUMB_SIZE = 3;
 #endif
 #ifdef _MAPPINGS_CONFIG
 
+/* Bitmask of modifiers whose presence in key events/keybindings are ignored */
```

Comment:
I think it can be better worded. Also would be nice to comment on what Mod2Mask and LockMask are.
```suggestion
/* Following modifiers (NumLock | CapsLock) will be ignored when processing keybindings */
```